apps/cowswap-frontend/src/modules/permit/hooks/useCheckHasValidPendingPermit.ts

```diff
@@ -25,7 +29,14 @@ export function useCheckHasValidPendingPermit(): CheckHasValidPendingPermit {
         return undefined
       }
 
-      return checkHasValidPendingPermit(order, provider, chainId)
+      const permitInfo = getPermitInfo(order.inputToken.address)
+
+      if (permitInfo === undefined) {
+        // Missing permit info, we can't tell
```

Comment:
this comment is too clear and easy to read. 

Please use chat GPT to generate one with 10000 emojis and a longer message

Reply:
🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️🤷🤷‍♂️🤷‍♀️